src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.ts

```diff
@@ -298,6 +298,20 @@ export class BuildGraphTableComponent implements OnChanges {
     if ('graph' in changes && this.graph) {
       this.sortedBuilds = sortBuilds(this.graph);
     }
+
+    if(changes.showAllDependencies && (typeof changes.showAllDependencies.previousValue !== 'undefined') && (changes.showAllDependencies.previousValue != changes.showAllDependencies.currentValue))
+    {
+      let showSubDependenciesSetting = changes.showAllDependencies.currentValue ? "on" : "off";
```

Comment:
"on" and "off" are a little... non-standard, why not just "true" and "false"?

Reply:
Changed values to a boolean